### PR TITLE
Cfp committee pics + cards

### PR DESCRIPTION
**Pull request by apiraino**
- [x] Added cards to CfP committee
- [x] Reduced images size
- [x] Missing info for Wouter Geraedts

### CODE REVIEW

**Comment by apiraino** [comment]
Screenshot_2019-08-22 CFP Committee

File: cfp-commitee.md [hunk 1]
```diff
@@ -63,3 +63,5 @@ Submit to the CFP
     {% endfor %}
   </ul>
 </section>
+
+<div>Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</a> is licensed by <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank">CC 3.0 BY</a></div>
```
Comment (apiraino) [inline cfp-commitee.md]: credited Flaticon for the placeholder

**Comment by djc** [comment]
Wouter Geraedts:

```yaml
  - icon: github
    link: https://github.com/Wassasin/
```

For me, can we add the following:

```yaml
  - icon: homepage
    link: https://dirkjan.ochtman.nl/
  - icon: twitter
    link: https://twitter.com/djco
  - icon: linkedin
    link: https://www.linkedin.com/in/dochtman/
```

**Comment by djc** [comment]
LGTM, for what it's worth!

**Comment by djc** [comment]
One question: I usually think of the "Call for Proposals" (CfP) preceding the selection of proposals into a talk program, and the committee that does that work is, as far as I've seen, more commonly called the Program Committee (rather than the CfP committee). Maybe it makes sense to change this?

**Comment by apiraino** [comment]
> One question: I usually think of the "Call for Proposals" (CfP) preceding the selection of proposals into a talk program, and the committee that does that work is, as far as I've seen, more commonly called the Program Committee (rather than the CfP committee). Maybe it makes sense to change this?

I have no specific knowledge on this, so I welcome suggestions to improve wording :+1:

**Comment by RafaelaWetzel** [comment]
Hey @apiraino could you use this profile picture for the CfP commitee instead, and also implement it on the RustFest about page?
Portrait

**Comment by RafaelaWetzel** [comment]
Hey @apiraino could you use this profile picture for the CfP commitee instead, and also implement it on the RustFest about page?
20190609-IMGP3550

File: _people/nuria.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+name: Núria
```
Comment (nuria-fl) [inline _people/nuria.md]:
```suggestion
name: Núria Soriano
```

File: _people/nuria.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+groups:
+  - committee
+links:
+  - icon: twitter
+    link: https://twitter.com/pincfloit
+
```
Comment (nuria-fl) [inline _people/nuria.md:12]:
```suggestion
  - icon: github
    link: https://github.com/nuria-fl
  - website:
    link: https://www.nuriasatorres.com/

```

**Review by nuria-fl — COMMENTED** [review]
Hi! Just a couple of changes on my details. Thanks!

**Pull request merged**